ChangeLog.md

```diff
@@ -1,5 +1,8 @@
 # eRecruiter API ChangeLog
 
+## 1.45.0
+- New property `Geolocation` of type `JobGeoLocationResponse` in JobResponse  which contains information about the job geolocation.

```

Comment:
Property is named `GeoLocation` in the class please correct to match the name in the class.

Reply:
done.